src/NuGet.Services.AzureSearch/DocumentUtilities.cs

```diff
@@ -0,0 +1,216 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Web;
+using NuGet.Frameworks;
+using NuGet.Protocol.Catalog;
+using NuGet.Services.Entities;
+using NuGet.Services.Metadata.Catalog;
+using NuGet.Versioning;
+using PackageDependency = NuGet.Protocol.Catalog.PackageDependency;
+
+namespace NuGet.Services.AzureSearch
+{
+    internal static class DocumentUtilities
+    {
+        private static readonly DateTimeOffset UnlistedPublished = new DateTimeOffset(1900, 1, 1, 0, 0, 0, TimeSpan.Zero);
```

Comment:
Can you reference `NuGet.Services.Metadata.Catalog.Constants.UnpublishedDate` instead of redefining this constant?

Reply:
That's a `DateTime`, but I'll re-use it for another static.